test/rsm/test.ts

```diff
@@ -1,44 +1,41 @@
 // See README.md for prerequisites for this to run
 
-const { Orchestrator, Config } = require('../../src')
-
-const testDna = Config.dna("test.dna.gz")
-
-const config = Config.gen({
-  tester: testDna,
-})
+import { Orchestrator, Config, InstallHapps } from "../../src"
+import { ScenarioApi } from "../../src/api"
+import path from 'path'
 
 const orchestrator = new Orchestrator()
 
-orchestrator.registerScenario('list dnas', async (s, t) => {
-  const { antony } = await s.players({ antony: config })
-  await antony.spawn()
-
-  const dnas = await antony.admin().listDnas()
-  console.log('dnas', dnas)
-
-  t.equal(dnas.length, 1)
-})
-
-orchestrator.registerScenario('call zome', async (s, t) => {
-  const { antony } = await s.players({ antony: config })
-  await antony.spawn()
-
-  const result = await antony.call('tester', 'foo', 'foo', { anything: 'goes' })
-  console.log('result', result)
-
-  t.equal(result, 'foo')
-})
-
-orchestrator.registerScenario('state dump', async (s, t) => {
-  const { antony } = await s.players({ antony: config })
-  await antony.spawn()
-
-  const dump = await antony.stateDump('tester')
-  t.equal(dump.length, 3)
-  t.ok(typeof dump[0].element === 'object')
-  t.ok(typeof dump[1].element === 'object')
-  t.ok(typeof dump[2].element === 'object')
+const conductor1Config = Config.gen()
+const conductor2Config = Config.gen()
+
+const conductor2Happs: InstallHapps = [
+  // agent 0 ... allow for length 1 lists, without brackets, to make it cleaner
+  path.join(__dirname, 'test.dna.gz'),
+  // agent 1
+  [path.join(__dirname, 'test.dna.gz')],
+  // agent 2
+  // [path.join(__dirname, 'test2.dna.gz')]
+]
+
+// orchestrator.registerScenario('list dnas', async (s: ScenarioApi, t) => {
+//   const [[antonyConductor, [[antony1TestCell], [antony2TestCell]]]] = await s.players(
+//     [[conductor2Config, conductor2Apps]]
+//   )
+//   const res = await antony1TestCell.call('foo', 'foo', null)
+//   // console.log('result!', res)
+//   t.equal(res, 'foo')
+// })
+orchestrator.registerScenario('list dnas', async (s: ScenarioApi, t) => {
+  const [player2] = await s.players([conductor2Config])
+  const [player2happ1, player2happ2] = await player2.installHapps(conductor2Happs)
+  const [player2happ2cell1] = player2happ2.cells
+  const res = await player2happ2cell1.call('foo', 'foo', null)
+  // or 
+  // const res = await player2happ2.cells[0].call('foo', 'foo', null)
```

Comment:
@maackle @JettTech @zippy 
the above is representing my latest thinking. This code is working. 

It's quite dynamic for what it allows, since an `App` can either be an array of DnaPath, or just a single DnaPath, meaning that the code can either be really clean when the case is simple, but allow for greater complexity and configuration. And the Typescript types will really help support developers to know about that, and how to use it. 

I got feeling the `Initialization` scheme was just too overloaded, just expose the `installHapps` function and expect its usage, which is simple, in all cases.  You can still auto-spawn/startup either all (the default), or none (the override), of the conductors, by passing a boolean as the second arg to `s.players`

Reply:
if you want control over the agent/key, you can use `player2.installHapp` whose second argument is an optional `AgentPubKey`
as you can see, `installHapps` is a very thin wrapper around that function anyways...
```
/**
   * helper to create agent pub keys and install multiple apps for scenario initialization
   */
  installHapps = (happs: InstallHapps): Promise<InstalledHapps> => {
    this._conductorGuard(`Player.installHapps`)
    return Promise.all(happs.map(h => this.installHapp(h)))
  }
```